nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -568,7 +568,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   The `nimPackages` and `nim2Packages` sets have been removed.
   See https://nixos.org/manual/nixpkgs/unstable#nim for more information.
 
-- [TODO: reword to place an attribute at the front] Programs written in [D](https://dlang.org/) using the `dub` build system and package manager can now be built using `buildDubPackage` utilizing lockfiles provided by the new `dub-to-nix` helper program.
+- `buildDubPackage` can now be used to build Programs written in [D](https://dlang.org/) using the `dub` build
+  system and package manager and utilizing lockfiles provided by the new `dub-to-nix` helper program.
```

Comment:
```suggestion
- `buildDubPackage` can now be used to build Programs written in [D](https://dlang.org/) using the `dub` build system and package manager.
```

I think there's no need to describe how lock files are created here, it is [in the manual](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/dlang.section.md#lockfiles-dub-lockfiles),